recipes/licensor/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "licensor" %}
+{% set version = "0.4.3" %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/jpvillaisaza/licensor/archive/refs/tags/v{{ version }}.tar.gz
+    sha256: 03bb7e621d43f88c3e4630b46a1bc80b71afe4f269cec2331cdca4c81bbce1bb
+
+
+build:
+  number: 0
+
+  missing_dso_whitelist:
+    - /usr/lib/libiconv.2.dylib  # [osx]
```

Comment:
This shouldn't be necessary, we should rather link to conda-forge's own `libiconv`.

Reply:
Could you explain to me how I would do that? @xhochy